app/src/androidTest/java/com/google/droidjump/LevelsScreenTest.java

```diff
@@ -0,0 +1,81 @@
+package com.google.droidjump;
```

Comment:
For tests, we try and maintain the following structure (basically each test has 3 blocks of code):
{
// setup of the test

// action which needs testing 

// checking that the behaviour obtained is expected
}

Reply:
Fixed